src/pkgcheck/checks/codingstyle.py

```diff
@@ -203,6 +267,9 @@ def feed(self, entry):
             yield MissingSlash(pkg, var, lines)
         for var, lines in unnecessary.items():
             yield UnnecessarySlashStrip(pkg, var, lines)
+        print(double_prefix)
```

Comment:
I imagine this is a errant debugging line.

Reply:
Sorry, removed now.